### ISSUE TITLE
rust on mac broken ?

### ISSUE THREAD

**Issue by lowks** [rust-lang/rust#23050]
I have installed the latest release nightly build:
```
➜  rusti git:(master) ✗ rustc --version
rustc 1.0.0-nightly (fed12499e 2015-03-03) (built 2015-03-03)
```

and this is the error I get:

```
rusti=> let xs: [i32; 5] = [1, 2, 3, 4, 5];
<anon>:13:5: 13:7 warning: unused variable: `xs`, #[warn(unused_variables)] on by default
<anon>:13 let xs: [i32; 5] = [1, 2, 3, 4, 5];
              ^~
LLVM ERROR: Program used external function '___morestack_addr' which could not be resolved!
An unknown error occurred

To learn more, run the command again with --verbose.
```
It has been like this since the previously nightly release.

**Comment by lowks** [comment]
This still seems to be happening with the new nightly release `2015-03-20`. Are there any workarounds for this ?
```
(item)➜  rusti git:(master) ✗ cargo run
   Compiling rusti v0.0.1 (file:///Users/lowks/Projects/src/rustlang/rusti)
     Running `target/debug/rusti`
rusti=> let xs: [i32; 5] = [1, 2, 3, 4, 5];
<anon>:13:5: 13:7 warning: unused variable: `xs`, #[warn(unused_variables)] on by default
<anon>:13 let xs: [i32; 5] = [1, 2, 3, 4, 5];
              ^~
LLVM ERROR: Program used external function '___morestack_addr' which could not be resolved!
An unknown error occurred

To learn more, run the command again with --verbose.
```

**Comment by murarth** [comment]
This issue has been fixed by #25627. This was confirmed with users of rusti (on murarth/rusti#14).
This issue can now be closed.
